### ISSUE TITLE
Qiskit installation error: cvxopt

### ISSUE THREAD

**Issue by ismaelfaro** [Qiskit/qiskit-metapackage#758]
When you install Qiskit in a Mac with Conda from scratch, the installation crash with this message:

    src/C/umfpack.c:23:10: fatal error: 'umfpack.h' file not found
    #include "umfpack.h"
             ^~~~~~~~~~~
    1 error generated.
    error: command 'gcc' failed with exit status 1

### Information

- **Qiskit Terra version**:  qiskit-0.14.0
- **Python version**: Python 3.8.0
- **Operating system**: OSX 10.15.2 [Catalina]

### What is the expected behavior?

The user can fix it installing: 

> conda install -c conda-forge cvxopt

### Suggested solutions

0) Minimum catch the error and show the user the way to fix it.
1) the better option is detect the dependence and install the Conda version.

**Comment by mtreinish** [comment]
I've moved this to the meta-repository because it is not a terra issue. cvxopt gets pulled in as a dependency and installed via the released versions of aer and aqua (and is an optional dep for the released version of ignis). cvxopt usage has been removed from master on ignis and aer, work is starting to do the same for aqua to remove it as a dependency there too. But, that doesn't actually solve the underlying issue here which is python 3.8 support doesn't yet exist for released versions of qiskit. Support for 3.8 has merged on master for terra and the ibmq provider, but PRs are still pending for aer (https://github.com/Qiskit/qiskit-aer/pull/428/ ), aqua (https://github.com/Qiskit/qiskit-aqua/pull/722 ), and ignis (https://github.com/Qiskit/qiskit-ignis/pull/298 ) which are all blocked by upstream dependencies. Until those have all merged and been included in a release `pip install qiskit` on python 3.8 won't just work and will required both a compiler and dev headers to able to build from source both qiskit and it's dependencies that don't have support yet.

That all being said this is also a duplicate of https://github.com/Qiskit/qiskit/issues/689

**Comment by mtreinish** [comment]
As for the suggest path forward, @1ucian0 is correct there is no way for qiskit to handle install time issues (as it's not installed). The fundamental issue here is that there are no precompiled python 3.8 binaries on pypi for qiskit's dependencies (and qiskit itself) which requires the user to manually install those dependencies using another tool (conda as suggested in the description, linux distro packaging, or brew on mac) or to manually install the build dependencies for everything missing a precompiled wheel before running pip. All of these complexities are why https://github.com/Qiskit/qiskit/pull/708 was added because the installation tooling  (mainly pip) doesn't know how to check all of this and has no mechanism to warn you that you'll have to compile a dependency from source prior to trying to install.

**Comment by mtreinish** [comment]
Qiskit 0.15.0 is out now and includes support for python 3.8. I'm going to close this issue now, please feel free to reopen it if I'm missing something.
